packages/generator/src/storybook.ts

```diff
@@ -20,12 +20,11 @@ const resourceTypes = [
   'Bot',
   'Project',
   'Communication',
-  'SpecimenDefinition',
```

Comment:
@rahul1 - It looks like `SpecimenDefinition` and`RequestGroup` were added to `storybook.ts`, but not actually executed, so the definitions were not in `packages/mocks/structuredefinitions.json`.  If they are needed, then we can add these back.

Reply:
I think they're needed for these stories: 
https://github.com/medplum/medplum/blob/main/packages/react/src/stories/ResourceTable.stories.tsx#L43-L47
https://github.com/medplum/medplum/blob/main/packages/react/src/stories/ResourceTable.stories.tsx#L43-L47

Reply:
I may have had a merge error on my previous PR that removed them from structuredefinitions.json.

Reply:
@codyebberson can you just check that those 2 stories work before merging?